### PR TITLE
Release v10.7.0

### PR DESCRIPTION
**Pull request by hobbitronics**
## Includes
- https://github.com/silinternational/ui-components/pull/242
- https://github.com/silinternational/ui-components/pull/241
- https://github.com/silinternational/ui-components/pull/240
- https://github.com/silinternational/ui-components/pull/239
- https://github.com/silinternational/ui-components/pull/238
- https://github.com/silinternational/ui-components/pull/237
- https://github.com/silinternational/ui-components/pull/234

## Added
- add height prop to StaticChip
- add inputID as prop on Checkbox
- deepfactor

## Removed
- sass from peer deps

## Fixed
- StaticChip: width now expands with content, styles overridable

## Updated
- dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Reusable Svelte components for some internal applications
...
+To install to your Svelte project, open your project's root directory in a terminal. Type or paste `npm i -D @silintl/ui-components` and press enter. Sass (depending on your bundler and plugins) and material-components-web are required to develop additional material web components in your app, but it may depend on your version of npm if they are necessary even if you don't (I haven't figured out which versions require them). Enter the commands `material-components-web@14` and `npm i -D sass@1` (versions will vary if you use an older release of this library). You should already have svelte@3 installed if you are using this library.
```
Comment (forevermatt) [inline README.md:7]: Is `material-components-web@14` really a command?

File: package.json [hunk 2]
```diff
@@ -31,33 +31,32 @@
...
     "@storybook/addon-links": "^6.5.16",
-    "@storybook/addon-postcss": "^2.0.0",
     "@storybook/addon-svelte-csf": "^2.0.11",
     "@storybook/builder-webpack5": "^6.5.16",
     "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/addon-postcss": "^2.0.0",
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/svelte": "^6.5.16",
+    "@semantic-release/changelog": "^6.0.2",
+    "@semantic-release/git": "^10.0.1",
```
Comment (forevermatt) [inline package.json:52]: Any idea why these were un-alphabetized?

**Review by forevermatt — APPROVED** [review]
Approved, with questions

File: README.md (hunk 1 above)
Comment (hobbitronics) [reply README.md:7]: Just a typo

File: package.json (hunk 2 above)
Comment (hobbitronics) [reply package.json]: Probably just got shuffled around during the scrutinizer work.

**Pull request merged**